src/components/ReferenceData.vue

```diff
@@ -208,18 +245,19 @@ export default {
 		},
 		async getAllReferenceData() {
 			this.isLoading = true
-			const res = await esApiSearchClient(this.esIndex, this.esDoctype, undefined, this.count)
+			const res = await esApiSearchClient(this.esIndex, this.esDoctype, "*" + this.esQueryExtra, this.count)
 			this.responseData = res.data
 			this.isLoading = false
 		},
 		async searchReferenceData(searchQuery) {
 			this.isLoading = true
-			const res = await esApiSearchClient(this.esIndex, this.esDoctype, searchQuery, this.count)
+			const res = await esApiSearchClient(this.esIndex, this.esDoctype, searchQuery + this.esQueryExtra, this.count)
 			this.responseData = res.data
 			this.isLoading = false
 		},
 		// TODO: if the es server is under too much stress debounce could be implemented
```

Comment:
this should be removed at this time as well

Reply:
not seeing any other comments about removing something, did you forgot to post some comment?

Reply:
oh, did you mean to delete this comment line?

Reply:
removed comment line